### PR TITLE
[Incomplete]Contact list implementation 1

### PR DESCRIPTION
**Pull request by yellowHatpro**
-Build Fails when clicking on the contacts button
-Logic to add contacts to be written yet

### CODE REVIEW

**Comment by SunitRoy2703** [comment]
> -Build Fails when clicking on the contacts button
> -Logic to add contacts to be written yet

Also, add the functionalities you have added in your PR

Follow this format, if you want to do this kind of prs

Done✅
- added.....

Yet to be done: 
- .....

**Comment by yellowHatpro** [comment]
> > -Build Fails when clicking on the contacts button
> > -Logic to add contacts to be written yet
> 
> Also, add the functionalities you have added in your PR
> 
> Follow this format, if you want to do this kind of prs
> 
> Done✅
> 
> * added.....
> 
> Yet to be done:
> 
> * .....

Ok Sir

**Comment by yellowHatpro** [comment]
Done ✅:
- Contacts entity ,Contacts dao ,Contacts db, Contacts repository ,Contacts ViewModel 
- Adapter for Recycler View to show contacts
- Corresponding design for RV
Yet to be done :
-Error Fixes
- Logic to add Contacts , using intent

File: app/src/main/java/org/openlake/sampoorna/viewmodel/ContactsViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.openlake.sampoorna.models.Contacts
+import org.openlake.sampoorna.models.ContactsDatabase
+import org.openlake.sampoorna.models.ContactsRepository
+
+class ContactsViewModel(application: Application) :AndroidViewModel(application) {
```
Comment (akshaaatt) [inline app/src/main/java/org/openlake/sampoorna/viewmodel/ContactsViewModel.kt:13]: Use Hilt implementation. That will ease your life.

**Review by akshaaatt — CHANGES_REQUESTED** [review]
There's a lot to unpack here.

**Pull request closed without merging**